test/riak_kv_sweeper_SUITE.erl

```diff
@@ -207,16 +309,62 @@ test_scheduler_worker_process_crashed(Config) ->
     ok = check_sweeps_idle(Indices),
     ok.
 
+%%
+%% Test riak_kv_sweeper's re-scheduling of sweeps, configured by the
+%% run_interval property. run_interval is an integer.
+%%
+%% The test uses sweep participant sweeper_callback_1 that completes
+%% successfully as soon as it is scheduled.
+%%
+%% The test asserts receipt of successful_sweep messages after sending
+%% sweep_ticks and then waiting run_interval+1 seconds, and then
+%% re-sending sweep_tick's.
+%%
 test_scheduler_run_interval(Config) ->
     Indices = ?config(vnode_indices, Config),
-    new_sweep_participant(sweeper_callback_1, 1),
+    RunInterval = 1,
+    new_sweep_participant(sweeper_callback_1, RunInterval),
+    sweeper_meck_helper:backend_modules(async, _NumKeys=1000, _ObjecSizeBytes=0),
+
+    sweep_all_indices(Indices),
+    timer:sleep(timer:seconds(RunInterval + 1)),
```

Comment:
It's not a new test, just looks like that in the diff. I'm not concerned about stability because the sleep is used to guarantee that a sweep is expired, ie., `os:timestamp()` after the sleep includes the slept amount, and not wait for receipt of messages. 

I think a mock would be a good way to eliminate the `timer:sleep(...)` and will try it out.